cmd/upload-s3/upload-s3.go

```diff
@@ -32,23 +31,13 @@ func main() {
 		grip.EmergencyFatal("remote path is missing")
 	}
 
-	awsKey := os.Getenv("AWS_KEY")
```

Comment:
will we be removing these from admin settings? Can we remove the fields entirely as a followup?

Reply:
These should get removed from Evergreen's project variables once this is deployed.

Could you explain what you mean by remove the fields entirely?

Reply:
Ah, yeah those are separate and will still exist (since we use them on the task hosts for task sync and writing logs).